packages/terra-hyperlink/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Fixed
+* Fixed the color contrast axe violation for disabled hyperlinks by adding `aria-disabled`.
```

Comment:
Was there an axe test for this?

Reply:
Good call @emilyrohrbough, we should add a test verifying this.

Reply:
Yes, there is an axe test for this but the color contrast rule was disabled.
So, I just enabled it now as seen here https://github.com/cerner/terra-core/pull/2730/commits/edb92facb331e3a1c7205538d5a5e90a9bcfcb81